python/jupytercad_app/package.json

```diff
@@ -23,7 +23,7 @@
   "style": "style/index.css",
   "repository": {
     "type": "git",
-    "url": "https://github.com/jupytercad/jupytercad.git"
+    "url": "https://github.com/jupytercad/jupyterCAD.git"
```

Comment:
> ValueError: package.json for '@jupytercad/jupytercad-app' does not define a 'repository.url' matching the cloned repository 'jupytercad/JupyterCAD'.

This error looks weird

Reply:
It seems you have a missing capital letter `jupyterCAD` -> `JupyterCAD`

Reply:
ah thanks! case-sensitive repo name never is a good idea